_data/fa-owners.yml

```diff
@@ -0,0 +1,353 @@
+'API definition':
+  - area: 'API definition'
+    label: api-definition
+    owner:
+    - sq-lb-epitome
+    repos:
+    - loopback-api-definition
+    - loopback-component-explorer
+    - loopback-swagger
+
+'Boot':
+  - area: Boot
+    label: boot
+    owner:
+    - sq-lb-asteroid
+    repos:
+    - loopback-boot
+    - loopback-example-app-logic
+    smes:
+    - davidcheung
```

Comment:
TY. ;)

I've been wanting to merge it forever, but @cgole @0candy @bosco-li are on point for this now as other parties need to be involved.

I personally agree with you on just landing this and changing it up when we receive confirmation. @cgole Is this possible?